settings.py

```diff
@@ -54,10 +54,15 @@
 # will be the path to the `stylus` and `lessc` executables.
 STYLUS_BIN = os.getenv('STYLUS_BIN', path('node_modules/stylus/bin/stylus'))
 LESS_BIN = os.getenv('LESS_BIN', path('node_modules/less/bin/lessc'))
-CLEANCSS_BIN = os.getenv('CLEANCSS_BIN',
-                         path('node_modules/clean-css/bin/cleancss'))
-UGLIFY_BIN = os.getenv('UGLIFY_BIN',
-                       path('node_modules/uglify-js/bin/uglifyjs'))
+CLEANCSS_BIN = os.getenv(
+    'CLEANCSS_BIN',
+    path('node_modules/clean-css/bin/cleancss'))
```

Comment:
I was going to say the opposite, I'm not sure this indentation is the best. It's not like it matters that much, but wonder if we can just agree on a set of rules, and then maybe "enforce" it (using flake8 or pyformat or anything like that) so we don't have to wonder each time if it's coherent with the rest of the code or not ;)